sport/app/rugby/feed/rugbyOptaDeserialisation.scala

```diff
@@ -255,7 +255,7 @@ object Parser {
   }
 
   private def chainAttributes(attributes: Iterable[MetaData]): MetaData = {
-    attributes match {
```

Comment:
Yes. But a case like `head :: tail` is not `Seq` - based either. So I decided not to use `Seq()`, because it would only work in situations when the pattern match doesn't involve head-tail matching.